docker/nginx.conf

```diff
@@ -6,6 +6,7 @@ server {
 
     ssl_certificate /etc/ssl/certs/ssl-cert.pem;
     ssl_certificate_key /etc/ssl/private/ssl-cert.key;
+    error_page 497 301 =307 https://$host:$server_port$request_uri;
```

Comment:
since this is typically for local hosting, i don't want to permanently affect browser cache (eg if hosting on localhost and tomorrow someone wants to use port 3000 for http, they dont want to be stuck with the cached redirect)